spec/js/collections/todos_spec.js

```diff
@@ -0,0 +1,37 @@
+define([
+  'models/todo',
+  'collections/todos'
+], function (ToDo, ToDoCollection) {
+  var instance, subject;
+
+  beforeEach(function () {
```

Comment:
This specs makes little sens, you are testing the framework, you can safely assume that the framework has it's own tests.